swiss_tournament/commands.py

```diff
@@ -0,0 +1,91 @@
+from typing import List, Optional
+
+from swiss_tournament.data.player import Player
+from swiss_tournament.data.round_pairing import RoundPairing
+from swiss_tournament.data.tournament import Tournament
+from swiss_tournament.parse.round_parser import RoundParser, YamlRoundParser
+from swiss_tournament.export.standings_exporter import MarkdownStandingsExporter, StandingsExporter
+from swiss_tournament.step.standings_generator import StandingsGenerator
+from swiss_tournament.export.tournament_exporter import TournamentExporter, YamlTournamentExporter
+from swiss_tournament.parse.tournament_parser import YamlTournamentParser, TournamentParser
+from swiss_tournament.export.round_exporter import YamlRoundExporter, RoundExporter, MarkdownRoundExporter
+from swiss_tournament.step.round_generator import RoundGenerator
+from swiss_tournament.step.tie_breaker import Buchholz
+from swiss_tournament.step.tournament_updater import TournamentUpdater
+
+
+def _get_class(kls: str):
+    parts = kls.split('.')
+    module = ".".join(parts[:-1])
+    m = __import__(module)
+    for comp in parts[1:]:
+        m = getattr(m, comp)
+    return m
+
+
+DEFAULT_TIE_BREAKERS = ["Buchholz", "MedianBuchholz", "Cumulative"]
```

Comment:
Can we actually set the default tie breaker to None? Sometimes we would want the type of tiebreaker to NOT influence the final standings, and so a tie in the final standings could be acceptable. It feels somehow restrictive otherwise.

Reply:
No problem here too.